packages/ui-extensions/docs/surfaces/point-of-sale/reference/targets/pos.purchase.post.action.menu-item.render.doc.ts

```diff
@@ -21,6 +21,10 @@ const data: ReferenceEntityTemplateSchema = {
       name: ExtensionTargetType.PosPurchasePostActionRender,
       url: '/docs/api/pos-ui-extensions/targets/pos-purchase-post-action-render',
     },
+    {
+      name: ExtensionTargetType.PosPurchasePostBlockRender,
```

Comment:
This is adding the new block target to the related section of the extisting menu item target.

Reply:
Hmm, interesting. Are these actually related? There's no way to communicate between them... But they're on the same screen. I don't know!

Reply:
Yeah menu-item doesn't seem related in the technical sense, only surface level